guide/using-python.md

```diff
@@ -17,22 +17,18 @@ On Ubuntu 16.04, it also supports Python 3.5 and on Ubuntu 14.04, it also suppor
+
 #### macOS and Linux installation
 
-Most of the Linux distributions have Python 2.7 and 3.x already installed.
-macOS has Python 2.7 installed by default.
+Most of the Linux distributions have Python 2.7 and 3.x already installed. macOS has Python 2.7 installed by default.
 It is possible to install Python 3.6 from the [Python web site](https://www.python.org) or from various Linux repositories.
-To check the current version of Python installed on your system, you can type in a terminal:
-
-```sh
-$ python --version
-```
-
-Webots will start Python using the standard `python` command line. As a consequence, it will execute the first `python` binary found in the current `PATH`. If you need a different version of Python, please install it if needed and configure your environment so that it becomes the default `python` version when called from the command line in a terminal.
+To check the versions of Python installed on your system, you can type in a terminal:
+`python --version`, `python3.6 --version`, `python2.7 --version`, `python3 --version`, etc.
 
 #### Windows installation
 
-You should install the latest version of Python 3.6 (64 bit) or Python 2.7 (64 bit) from the official [Python website](https://www.python.org). Then, you have to modify your `PATH` environment variable to add the path to the python.exe binary which is located in the main `Python36` or `Python27` installation folder. To check this was done properly, you can open a DOS console (CMD.EXE) and type `python --version`. If it displays the correct Python version, then, everything is setup properly and you should be able to run the Python example provided with Webots in the `WEBOTS_HOME/projects/languages/python/worlds/example.wbt` world file.
+You should install the latest version of Python 3.6 (64 bit) or Python 2.7 (64 bit) from the official [Python website](https://www.python.org). Then, you have to modify your `PATH` environment variable to add the path to the `python.exe` binary which is located in the main `Python36` or `Python27` installation folder. To check this was done properly, you can open a DOS console (`CMD.EXE`) and type `python --version`. If it displays the correct Python version, then, everything is setup properly and you should be able to run the Python example provided with Webots in the `WEBOTS_HOME/projects/languages/python/worlds/example.wbt` world file.
```

Comment:
Personally, I think that this line is too long and shuld be splitted on multiple lines.
Identifying the changes you made is really difficult.